python/LoadStac/load-stac-item-example.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create example GTiff\n",
+    "First we download a simple GTiff file containing ndvi data from an openeo provider."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openeo\n",
+    "import pystac\n",
+    "from datetime import datetime\n",
+    "import rasterio\n",
+    "import rasterio.warp\n",
+    "from rasterio.plot import show"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Authenticated using refresh token.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<Connection to 'https://openeo.dataspace.copernicus.eu/openeo/1.2/' with OidcBearerAuth>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# connection = openeo.connect(\"https://openeo.vito.be\")\n",
+    "connection = openeo.connect(\"openeo.dataspace.copernicus.eu\")\n",
+    "connection.authenticate_oidc()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bbox = {\"west\": 5.0, \"south\": 51.2, \"east\": 5.1, \"north\": 51.3}\n",
+    "collection = connection.load_collection(\n",
+    "    \"SENTINEL2_L2A\",\n",
+    "    spatial_extent=bbox,\n",
+    "    temporal_extent=[\"2021-06-01\", \"2021-06-30\"],\n",
+    "    bands=[\"B04\", \"B08\", \"SCL\"],\n",
+    "    max_cloud_cover=95\n",
+    ")\n",
+    "collection = collection.process(\"mask_scl_dilation\", scl_band_name=\"SCL\", data=collection)\n",
+    "collection = collection.ndvi(red=\"B04\",nir=\"B08\", target_band=\"NDVI-band\").filter_bands(\"NDVI-band\")\n",
+    "collection = collection.reduce_temporal(\"mean\")\n",
+    "collection.download(\"ndvi-file.tiff\", format=\"GTiff\")"
```

Comment:
In synchronous, specifying the output format might not be needed since you already define the output with the format.